### ISSUE TITLE
Chef quote style changes

### ISSUE THREAD

**Issue by caitlynmayers** [caitlynmayers/dukes#143]
### Location
http://test-dukes.pantheonsite.io/

### Description
Please change the background color of the homepage chef quote area to #fc611e, and increase the text size if the attribution to 18px.

**Comment by caitlynmayers** [comment]
So this is still coming back as not enough contrast, but according to the tools, the colors pass the AA Large text requirement, so I am leaving this style as is for now.

**Comment by caitlynmayers** [comment]
@hessdev this is still failing, and it's my fault for misinterpreting the guidelines. Please change the type size of the quote p to 24px, and change the type size of the chef meta to 20px and this should pass. Thanks!

**Comment by caitlynmayers** [comment]
Fix verified.
